guides/source/testing.md

```diff
@@ -423,7 +505,7 @@ though:
 class MyTest < ActiveSupport::TestCase
   test "current_transaction" do
```

Comment:
To separate the Active Record method name from the test case name, this might be clearer:

```suggestion
  test "Active Record current_transaction method works as expected" do
```

Also we should link the ActiveRecord::Base.current_transaction method.